spec/validators/place_bid_validator_spec.rb

```diff
@@ -32,7 +32,8 @@
       PlaceBidValidator.new.validate(bid)
 
       expect(bid.errors.full_messages).to include(
-        I18n.t('activerecord.errors.models.bid.amount.greater_than_max')
+        I18n.t('activerecord.errors.models.bid.amount.greater_than_max',
```

Comment:
super tiny formatting thing: with multiple args, I usually put each one on its own line if they are being separated:


```
I18n.t(
  'activerecord.errors.models.bid.amount.greater_than_max',
    max_allowed_bid: 149
)
```

I feel it helps with readability. What do you think?